IntroToCSharp/Pages/Lessons/Collections/Arrays.razor

```diff
@@ -0,0 +1,379 @@
+@inherits CaptainCoder.Page
+@page "/lessons/collections/arrays"
+
+<Title>Arrays</Title>
+
+
+<Section Title="Introdution">
+    <MudText>
+        An array is a data structure that stores a pre-defined number of variables of the same data type within its bounds.
+         The important part of this definition to take note of is that an array is predefined, meaning that you have to know 
+         how many variables are going to be within the array before you create it.
+         This is because arrays are immutable data structures which means after they are created their <b>length</b> can't be changed.
+    </MudText>
+
+    <Section IsSubSection="true" Title="The Rules">
+        <MudText>When dealing with an array there are four main things to keep in mind:</MudText>
+        <InfoGrid>
+            <GridSquare>Arrays have a <b>pre-defined</b> length. You must give an array a length before you are able to implement it.
+            </GridSquare>
+            <GridSquare>Arrays are <b>immutable</b>. Once the length an array is set, you cannot change it.
+            </GridSquare>
+            <GridSquare>Arrays <b>cannot</b> mix variables types.
+            </GridSquare>
+            <GridSquare>An array's index always begins at 0.
+            </GridSquare>
+        </InfoGrid>
+    </Section>
+</Section>
+
+<Section Title="Our First Array">
+    <MudText>
+        Let's start by making our first array! To do this we are going to start with a string of heroes.
+        To make an array we have to start by declaring what data type that the array will be using followed by
+        a pair of brackets (<Code>[]</Code>) and finally the array's name.
+    </MudText>
+    <MudText>
+        When we set an array's value we "wrap" the value within a pair of curly brackets (<Code>[]</Code>) and seperate
+        each variable with a comma, like so:
+    </MudText>
+
+    <CodeBlock>
+        string[] justiceLeague = {"Superman", "Batman", "Aquaman", "Green Lanter", "Flash", "Wonder Woman", "Martian Manhunter"};
+    </CodeBlock>
+
+    <MudText>
+        As you can see we have a string of 7 superheroes within our array that make up the original Justice League! Each variable
```

Comment:
Try to avoid statements like "As you can see". This type of phrase can make someone who doesn't see it right away feel dumb.

Insteady try something like:
"In the code above we initialize the variable to reference an array with 7 strings in it."

Reply:
That's a good point. Thanks